app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -207,7 +207,7 @@ class Exposure extends React.Component {
         ...staticValidations,
         symptom_onset: yup
           .date('Date must correspond to the "mm/dd/yyyy" format.')
-          .max(moment().add(30, 'days').toDate(), 'Date can not be more than 30 days in the future.')
+          .max(moment().toDate(), 'Date can not be in the future.')
```

Comment:
I think some sort of additional validation should exist on the "Symptom Onset" field in the Reports table. 
![image](https://user-images.githubusercontent.com/52747882/127225372-d6f3ed60-9927-46c7-b859-cb5f7e9fcffa.png)
Currently if I put a future symptom onset in there, the model validation catches it, but not very elegantly:
![image](https://user-images.githubusercontent.com/52747882/127225436-cb7e3210-7e2d-4cc5-9c12-bb314f9ca605.png)

Reply:
good catch: https://github.com/SaraAlert/SaraAlert/pull/1018/commits/facaa28644b2c6177d1834217ba8ba60c6c62a06